src/Compilers/CSharp/Test/Semantic/Semantics/RecordTests.cs

```diff
@@ -29918,5 +30135,106 @@ class C2 : I(0)
                 Diagnostic(ErrorCode.ERR_UnexpectedArgumentList, "(0)").WithLocation(10, 13)
                 );
         }
+
+        [Theory, WorkItem(44902, "https://github.com/dotnet/roslyn/issues/44902")]
+        [CombinatorialData]
+        public void CrossAssemblySupportingAndNotSupportingCovariantReturns(bool useCompilationReference)
```

Comment:
@AlekseyTs Would you mind looking at this test? Couldn't get rid of the extra diagnostics (WRN_UnifyReferenceMajMin).

Reply:
> Would you mind looking at this test? Couldn't get rid of the extra diagnostics (WRN_UnifyReferenceMajMin).

It is probably unavoidable given the current set of reference assemblies. I suggest filtering them out 
or suppressig in compilation options to reduce the noise.